test/python/test_user_config.py

```diff
@@ -25,87 +23,83 @@
 
 class TestUserConfig(QiskitTestCase):
 
+    @classmethod
+    def setUpClass(cls):
+        cls.file_path = 'temp.txt'
+
+    def tearDown(self):
```

Comment:
I normally use `addCleanup()` instead of `tearDown()` for this kind of thing then we don't need to use the class to store a common file name. But this works fine too.

Reply:
I felt that it would be cleaner to have `self.file_path` everywhere rather than random strings. I will update it to use `addCleanup()` instead though.